GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2179,6 +2181,8 @@ protected override bool ExecuteCommand(int cmd)
                 case Commands.OpenWithDifftool: OpenWithDifftool(); break;
                 case Commands.OpenSettings: OnShowSettingsClick(null, null); break;
                 case Commands.ToggleBranchTreePanel: toggleBranchTreePanel_Click(null, null); break;
+                case Commands.GoToSuperproject: toolStripButtonLevelUp_ButtonClick(null, null); break;
```

Comment:
The commands were just sorted, suggest these are added in order too

Reply:
I don't see them being sorted.

Reply:
#5372 is not merged yet

Reply:
You have tried that branch - this the reason you have already assigned these hotkeys (they are identified in settings by the same numbers)

Reply:
I changed the numbers to avoid a conflict with https://github.com/gitextensions/gitextensions/pull/5372 numbers